modules/jtframe/target/mist/cfgstr

```diff
@@ -4,7 +4,8 @@
 {{ if .JTFRAME_OSD_LOAD }}          F,rom; {{ end }}
 {{ if .JTFRAME_VERTICAL }}
 {{ if .JTFRAME_OSD_FLIP }}          O1,Flip screen,Off,On; {{ end }}
-                                    O2,Rotate controls,Yes,No; {{ end }}
+                                    O2,Rotate controls,Yes,No;
+{{ if .JTFRAME_SDRAM_ROTATION }}    OD,Screen Rotation,No,Yes; {{ end }}{{ end }}
```

Comment:
On MiSTer, there is a way to disable the display of this line when the game is not vertical. So a core that handles both vertical and horizontal games only displays it when needed.

Reply:
There's no feedback to the OSD from the core in the MiST firmware. It won't be hard to do it, just the urge wasn't too strong to implement it.